collect_app/src/main/java/org/odk/collect/android/widgets/warnings/SpacesInUnderlyingValuesWarning.java

```diff
@@ -0,0 +1,116 @@
+package org.odk.collect.android.widgets.warnings;
+
+import android.content.Context;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.FluentIterable;
+import com.google.common.collect.Lists;
+import org.javarosa.core.model.SelectChoice;
+import org.odk.collect.android.R;
+import org.odk.collect.android.utilities.UnderlyingValuesConcat;
+import org.odk.collect.android.widgets.QuestionWidget;
+
+import java.util.List;
+
+public class SpacesInUnderlyingValuesWarning {
```

Comment:
A doc comment here would help readers understand what this is

Reply:
Added documentation in b488926